app/assets/json/translations/locale-en.json

```diff
@@ -739,29 +771,76 @@
                 "visit_a_housing_counselor": "Visit a local housing counselor for help with your application.",
                 "visit_homeownership_sf": "Visit HomeownershipSF"
             },
+            "at_total_income": "at <strong>{{income}}</strong> per {{per}}",
```

Comment:
@software-project I actually think you should include the per in the variable, and use the per_month and per_year strings that we already have translated as the timeframe variable instead of just year vs month